modules/azure/build.gradle

```diff
@@ -1,9 +1,15 @@
 description = "Testcontainers :: Azure"
 
+tasks.japicmp {
+    packageExcludes = [
+        "org.testcontainers.shaded.*",
```

Comment:
We should make this exclusion global in gradle/japicmp.gradle 👍

Reply:
done